web/src/components/tasks/business-formation/DbaFormationPaginator.tsx

```diff
@@ -161,7 +161,11 @@ export const DbaFormationPaginator = (): ReactElement => {
   };
 
   const displayButtons = (): ReactNode => {
-    if (state.stepIndex === 0 && state.businessNameAvailability?.status === "AVAILABLE") {
+    if (
+      state.stepIndex === 0 &&
+      ((isNotDba && state.businessNameAvailability?.status === "AVAILABLE") ||
+        (!isNotDba && state.dbaBusinessNameAvailability?.status === "AVAILABLE"))
```

Comment:
I gotta say, `!isNotDba` .... could be renamed I think

Reply:
Again, I think things like this would be easier to read if it was extracted into a variable. Like, we want to be able to quickly read a line and understand the if-case for these buttons, and the added complexity makes it hard.

Something like `if (state.stepIndex === 0 &&& nameIsAvailable)`

Reply:
renamed and replaced with `isDba`
broke out the status comparison